OpenEdXMobile/res/menu/delete_contextual_menu.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/item_delete"
+        android:icon="@drawable/ic_menu_delete"
+        android:title="@string/label_delete"
```

Comment:
My actual concern was this property `ifRoom|withText` , we should avoid to show text with button in action bar as we dont' have any such requirements.